<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWidget</class>
 <widget class="QWidget" name="CalibrationWidget">
  <property name="objectName" >
   <string notr="true" >CalibrationWidget</string>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="targetPointLabel">
     <property name="text">
      <string>Target point: 'Point'</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="targetImageLabel">
     <property name="text">
      <string>Target image coordinates: (u, v)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="toolheadPosLabel">
     <property name="text">
      <string>Current toolhead position: (x, y)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="upperLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Please move the toolhead to the specified point</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="lowerLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
